Code/src/main/java/nl/utwente/viskell/ghcj/GhciEvaluator.java

```diff
@@ -62,6 +62,12 @@ print a zero byte whenever it expects the user (that's us) to enter
            useful sentinel. */
         this.eval(":set prompt " + SENTINEL);
 
+        /* Load some useful libraries. */
+        this.eval("import Prelude");
+        this.eval("import Data.List");
+        this.eval("import Data.Maybe");
+        this.eval("import Data.Either");
```

Comment:
Making the list observable sounds a bit scary for something so deep inside the backend, but you're right that this should be a list inside GhciEvaluator. Will fix.